api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1073,7 +1072,7 @@ message Cluster {
   bool connection_pool_per_downstream_connection = 51;
 }
 
-// [#not-implemented-hide:] Extensible load balancing policy configuration.
+// Extensible load balancing policy configuration.
```

Comment:
Since these APIs weren't implemented and hided for long time, shall we update Policy to TypedExtensionConfig before it goes live? cc @envoyproxy/api-shepherds

Reply:
Definitely seems desirable. Can this work for gRPC @markdroth?

Reply:
I don't think gRPC is using this field yet, so I think that should be fine.  @ejona86 and @dfawley can correct me if I'm wrong.